### PR TITLE
Update Seafile license to reflect different licenses 

### PR DESCRIPTION
**Pull request by spbriggs**
Different components are under different licenses https://download.seafile.com/published/seafile-manual/home.md

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
I think your editor messes with a lot of the UTF-8 symbols in the file.

File: README.md [hunk 1]
```diff
@@ -893,11 +893,11 @@ _Games, game servers and control panels._
...
 - [openHAB](http://www.openhab.org) - Vendor and technology agnostic open source software for home automation. ([Source Code](https://github.com/openhab/openhab)) `EPL-1.0` `Java`
 - [Thingsboard](https://thingsboard.io/) - Open-source IoT Platform - Device management, data collection, processing and visualization. ([Demo](https://demo.thingsboard.io/signup), [Source Code](https://github.com/thingsboard/thingsboard)) `Apache-2.0` `Java`
-- [Thingspeak](https://thingspeak.com/) - Open source “Internet of Things” application and API to store and retrieve data from things using HTTP. ([Demo](https://thingspeak.com/channels/public), [Source Code](https://github.com/iobridge/thingspeak)) `GPL-3.0` `Ruby`
+- [Thingspeak](https://thingspeak.com/) - Open source "Internet of Things" application and API to store and retrieve data from things using HTTP. ([Demo](https://thingspeak.com/channels/public), [Source Code](https://github.com/iobridge/thingspeak)) `GPL-3.0` `Ruby`
```
Comment (SuperSandro2000) [inline README.md:900]: Every other character change except this one should be reverted.

File: README.md [hunk 2]
```diff
@@ -768,7 +768,7 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 #### Single-click/drag-n-drop upload
 
 - [BoZoN](https://github.com/broncowdd/BoZoN) - Minimalist Drag and drop file sharing app. `AGPL-3.0` `PHP`
-- [Coquelicot](https://coquelicot.potager.org/) - Coquelicot is a “one-click” file sharing web application with a focus on protecting users’ privacy. ([Source Code](https://coquelicot.potager.org/gitweb/?p=coquelicot.git)) `AGPL-3.0` `Ruby`
+- [Coquelicot](https://coquelicot.potager.org/) - Coquelicot is a "one-click" file sharing web application with a focus on protecting users' privacy. ([Source Code](https://coquelicot.potager.org/gitweb/?p=coquelicot.git)) `AGPL-3.0` `Ruby`
```
Comment (SuperSandro2000) [inline README.md:771]: and this

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by spbriggs** [comment]
I'll have another try and submit a different PR. Apologies.

**Comment by jnwatts** [comment]
It's generally better to [amend](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History) your commit and update the PR, rather than start a whole new one :-)

**Pull request closed without merging**